enterprise/backend/src/metabase_enterprise/advanced_permissions/models/permissions/group_manager.clj

```diff
@@ -23,14 +23,24 @@
   [user-group-memberships]
   (into {} (map (fn [x] [(:id x) (dissoc x :id)]) user-group-memberships)))
 
+(defn- complete-membership-info
+  "Fill in missing :is_group_manager values to be based on existing values, or 'false' if not set"
+  [new-user-info old-user-info]
+  (map (fn [x]
+         (if (nil? (:is_group_manager x))
```

Comment:
I think this would be clearer and more concise as:

```
(cond-> x
  (nil? (:is_group_manager))
  (assoc :is_group_manager (:is_group_manager (old-user-info (:id x)) false)))
```

You could also do `(boolean (:is_group_manager (old-user-info (:id x))))` if you think that's clearer - I think either one makes sense (`(boolean nil)` is `false`).

Reply:
Or actually maybe even `(assoc :is_group_manager (-> x :id old-user-info :is_group_manager (or false)))` would be the most readable.

Reply:
Applied your suggestions. Thanks!